assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-cta.scss

```diff
@@ -38,7 +52,11 @@
 
 	.googlesitekit-idea-hub__dashboard-cta__icon {
 		display: inline-block;
-		padding: $grid-gap-desktop 0 $grid-gap-desktop $grid-gap-desktop;
+		padding-bottom: $grid-gap-desktop;
+
+		@media (min-width: $bp-tablet) {
+			padding: $grid-gap-desktop 0 $grid-gap-desktop 0;
```

Comment:
Same, we don't need to add top and bottom padding since the elements are side by side. The paddings do not do anything here 😁